### PR TITLE
UDS-1768: feat(component-header): removed display:unset on Header hidden inputs

### PR DESCRIPTION
**Pull request by juanmitriatti**
UDS-1768

feat(component-header): removed display:unset on Header hidden inputs

### Links

- [JIRA ticket](https://asudev.jira.com/browse/UDS-1768)
- [Unity reference site](https://asu.github.io/asu-unity-stack/)
- [Unity Design Kit](https://shared-assets.adobe.com/link/fb14b288-bf63-47e0-5d30-384de5560455)

### FOR APPROVERS

- [Percy build approval](https://percy.io/5eae92d9/-all-UDS-packages)

### CODE REVIEW

File: packages/component-header/src/components/UniversalNavbar/Search/index.styles.js [hunk 1]
```diff
@@ -20,7 +20,6 @@ const SearchWrapper = styled.form`
     color: #495057;
     border: 1px solid #707070;
     margin: 0.5rem 0;
-    display: unset;
```
Comment (scott-williams-az) [inline packages/component-header/src/components/UniversalNavbar/Search/index.styles.js]: Looks good.
Can we also add some pattern to make sure it is only applying to search field?

Line 12 change `input` to:
`input[type="search"] ` or `input[name="q"]`

Comment (juanmitriatti) [reply packages/component-header/src/components/UniversalNavbar/Search/index.styles.js]: The inputs don't have the type, but I'll try adding them to see if it works.

Comment (scott-williams-az) [reply packages/component-header/src/components/UniversalNavbar/Search/index.styles.js]: The input should always have the name, so lets use `input[name="q"]` then

Comment (juanmitriatti) [reply packages/component-header/src/components/UniversalNavbar/Search/index.styles.js]: @scott-williams-az 
The problem are these inputs :
```
<input name="url_host" value={site} type="hidden" />
<input name="site" value="default_collection" type="hidden" />
<input name="sort" value="date:D:L:d1" type="hidden" />
<input name="output" value="xml_no_dtd" type="hidden" />
<input name="ie" value="UTF-8" type="hidden" />
<input name="oe" value="UTF-8" type="hidden" />
<input name="client" value="asu_frontend" type="hidden" />
<input name="proxystylesheet" value="asu_frontend" type="hidden" />
```
These were incorrectly read in Safari; the display: unset caused the tabbing through the site to pass through these input fields, which shouldn't happen.
So, the selector you suggested would not be enough : `input[name="q"]` .  I can use type=hidden as selector

Comment (scott-williams-az) [reply packages/component-header/src/components/UniversalNavbar/Search/index.styles.js]: Previously this code was targeting all the inputs, including the hidden inputs. 
```
input {
    width: 200px;
    ...
    display: unset;
```
This CSS rule is only meant for the one visible input, which is why I am suggesting we had `input[name="q"]`. We do not need to write any style rules for any hidden elements. By making this rule only apply to the visible input that also means we are not adding any styles (like a `display: unset`) to the hidden inputs.

Comment (juanmitriatti) [reply packages/component-header/src/components/UniversalNavbar/Search/index.styles.js]: @scott-williams-az  understood. Working on that fix.

Comment (juanmitriatti) [reply packages/component-header/src/components/UniversalNavbar/Search/index.styles.js]: Hi @scott-williams-az 
This is ready to test again.

**Review by mlsamuelson — APPROVED** [review]

**Pull request merged**